fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/TransactionService.java

```diff
@@ -31,6 +31,10 @@
 @Slf4j
 @RequiredArgsConstructor
 public class TransactionService {
+    //TODO Fix hardcoded values
+    private static final int PAGE = 1;
+    private static final int TRANSACTIONS_PER_PAGE = 500;
```

Comment:
Passing pageSize as null should force returning all (or provider default value - i.e. 500 for the certain provider) transactions. So this should be null and unnecessary

Reply:
Deleted unnecessary vars, changed fields to null.